pkg/storage/module/delete_test.go

```diff
@@ -0,0 +1,55 @@
+package module
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/gobuffalo/envy"
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/stretchr/testify/suite"
+)
+
+type DeleteTests struct {
+	suite.Suite
+}
+
+func TestDelete(t *testing.T) {
+	suite.Run(t, new(DeleteTests))
+}
+
+func (m *DeleteTests) TestDeleteTimeout() {
```

Comment:
I think it may be more idiomatic to use the first letter of the type. So here `(d *DeleteTests)` would be better.